### PR TITLE
Add platform check to PostBuild.cs

### PR DESCRIPTION
**Pull request by Sencerd**
Guard against missing dependency error when Unity iOS build support is not installed

### CODE REVIEW

File: Assets/Plugins/Appboy/Editor/PostBuild.cs [hunk 1]
```diff
@@ -1,10 +1,12 @@
-﻿using System;
+#if UNITY_IOS
+using System;
 using System.IO;
 using System.Text.RegularExpressions;
 using UnityEngine;
 using UnityEditor;
 using UnityEditor.Callbacks;
 using UnityEditor.iOS.Xcode;
```
Comment (Bucimis) [inline Assets/Plugins/Appboy/Editor/PostBuild.cs]: why gate all dependencies instead of just `using UnityEditor.iOS.Xcode;`?

Comment (Sencerd) [reply Assets/Plugins/Appboy/Editor/PostBuild.cs]: Because none of them are used unless you are on iOS. Either way is fine though.

Comment (Bucimis) [reply Assets/Plugins/Appboy/Editor/PostBuild.cs]: works for me

**Comment by Bucimis** [comment]
merged, thanks @Sencerd!

**Pull request merged**